DevOps Engineer/seo-dev-ops-engineer

```diff
@@ -0,0 +1,48 @@
+Job description
+
+We are looking for a DevOps Engineer to join our community of digital specialists to help deliver more great teachers
+and value for money services.
+You will work in a team of skilled DevOps engineers providing DevOps capability to the Department For Education’s
+Teacher Services division. You will be working with multi-disciplinary teams who deliver user centred digital services
```

Comment:
As this is a generic description, suggest we remove references to specific divisions, or put in a [division] placeholder?